app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
Still, I
a) do not use a separate td for the date. Then the dates will not be aligned any more and the more the lengths of the user names differ, the uglier the layout will be
b) close the previous td and open another td within the __() expression. Then the structure of the code will not reflect the structure of the HTML any more. 

Do you still prefer to include it directly, i.e. by using (a)?